src/chocolatey/infrastructure.app/services/RegistryService.cs

```diff
@@ -398,7 +398,7 @@ public Registry read_from_file(string filePath)
                 return null;
             }
 
-            return _xmlService.deserialize<Registry>(filePath);
+            return _xmlService.deserialize<Registry>(filePath, 1);

```

Comment:
So does this mean 1 to get it to try to run things at least one time?

Reply:
Or does 1 mean it will retry it once?

Reply:
When I tested it, this meant it will be executed once, and only once, and it wouldn't try it again.